dali/test/python/test_utils.py

```diff
@@ -164,3 +168,22 @@ def __next__(self):
         return (batch)
 
     next = __next__
+
+class DataIterator(object):
+    def __init__(self, batch_size, y_data, dtype):
+        self.batch_size = batch_size
+        self.test_data = []
+        for _ in range(self.batch_size):
+            self.test_data.append(np.array(y_data, dtype=dtype))
```

Comment:
Do we really wan tot have the same data across the samples?

Reply:
I renamed this as "ConstantDataIterator"